Add IQS050 rule (#2424)
home-assistant/developers.home-assistant: docs/core/integration-quality-scale/rules/iqs050.md

---

> title: "IQS050 - The documentation describes known limitations of the integrations (not to be confused with bugs)"
> ---
> 
> IQS050 is as follows:
> 
> The documentation describes known limitations of the integrations (not to be confused with bugs)

Reviewer: ```suggestion
title: "IQS050 - The documentation describes known limitations of the integration (not to be confused with bugs)"
---

IQS050 is as follows:

The documentation describes known limitations of the integration (not to be confused with bugs)
```

## Reasoning

> Users should know what to expect from an integration.
> So we want to make sure that the documentation describes known limitations of the integrations.

Reviewer: ```suggestion
Describing the known limitations of the integration in the documentation will allow users to know what to expect from the integration.
```

We should refrain from noting down bugs, since we use the issue tracker at GitHub for that and we don't want to duplicate information.

## Example implementation

```markdown
## Known limitations

> The integration does not provide the ability to reboot, which is available via the manufacturer's app.

Reviewer: ```suggestion
The integration does not provide the ability to reboot, which can instead be done via the manufacturer's app.
```
